ColorCode.WinUI/ColorCode.WinUI.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0-windows10.0.18362.0</TargetFramework>
```

Comment:
Is WinUI 3 min not 17763? Or is this the target? Does it matter that it's not newer?

Reply:
This is the target. Its actually better to target this, as we had problems in the past with targeting newer SDKs. I need to follow up with the C#/WinRT team on this, but for now this is the min we can target (not the min we support).